app/Service/ObjectRestrictorBuilder.php

```diff
@@ -28,6 +31,34 @@ public function prepareAccessibilityRestrictions(array $accessibility)
         return $restrictions;
     }
 
+    /**
+     * Pripravi restrikce pro pristupnost mapovych objektu.
+     *
+     * @param array $accessibility
+     *
+     * @return array
+     */
+    public function prepareAccessibilityPramRestrictions(array $accessibility)
+    {
+        $restrictions = ["[accessibility_pram_id] IN %in", $accessibility];
```

Comment:
U obou nově přidaných kategorií přístupnosti musí být ošetřeno, že pokud objekt nemá přístupnost nastavenu, použije se hodnota `accessibility_id`

Reply:
@vrbas a to do toho SQL zapíšu jak prosím tě?